tests/Behat/Context/Api/ChannelPricingLogEntryContext.php

```diff
@@ -0,0 +1,105 @@
+<?php
+
+/*
+ * This file is part of the Sylius package.
+ *
+ * (c) Paweł Jędrzejewski
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+declare(strict_types=1);
+
+namespace Tests\Sylius\PriceHistoryPlugin\Behat\Context\Api;
+
+use ApiPlatform\Core\Api\IriConverterInterface;
+use Behat\Behat\Context\Context;
+use Sylius\Component\Core\Formatter\StringInflector;
+use Sylius\Component\Core\Model\ProductInterface;
+use Sylius\Component\Core\Repository\ProductVariantRepositoryInterface;
+use Sylius\Component\Product\Resolver\ProductVariantResolverInterface;
+use Sylius1_11\Behat\Client\ApiClientInterface;
+use Sylius\Behat\Client\ResponseCheckerInterface;
+use Sylius\Behat\Service\SharedStorageInterface;
+use Sylius\Component\Core\Model\ProductVariantInterface;
+use Webmozart\Assert\Assert;
+
+final class ChannelPricingLogEntryContext implements Context
+{
+    public function __construct(
+        private ApiClientInterface $client,
+        private ResponseCheckerInterface $responseChecker,
+        private ProductVariantRepositoryInterface $productVariantRepository,
+        private SharedStorageInterface $sharedStorage,
+    ) {
+    }
+
+    /**
+     * @When I go to the :variantName product variant price history
+     */
+    public function iGoToTheVariantPriceHistory(string $variantName): void
```

Comment:
```suggestion
    public function iGoToTheVariantPriceHistory(ProductVariantInterface $variant): void
```
Is that not enough here (with proper adjusting the step of course)? If not, maybe we should set the name for the variant.